src/main/java/com/gudimov/movieland/controller/MovieController.java

```diff
@@ -59,5 +60,20 @@
         return new ResponseEntity<>(movieJson, HttpStatus.OK);
 
     }
+
+    @RequestMapping(value = "/genre/{genreId}")
+    @ResponseBody
+    public ResponseEntity<String> getMovieByGenreId(@PathVariable int genreId){

```

Comment:
getByGenre as method name will be better because you're currently in the Movie controller